datahub-web-react/src/app/entityV2/shared/EntityDropdown/CreateGlossaryEntityModal.tsx

```diff
@@ -58,6 +59,21 @@ function CreateGlossaryEntityModal(props: Props) {
     const [createGlossaryTermMutation] = useCreateGlossaryTermMutation();
     const [createGlossaryNodeMutation] = useCreateGlossaryNodeMutation();
 
+    useEffect(() => {
+        if (props.isCloning && entityData.entityData) {
+            const { properties } = entityData.entityData;
+
+            if (properties?.name) {
+                setStagedName(properties.name);
+                form.setFieldValue('name', properties.name);
+            }
```

Comment:
what do you think about doing `${properties.name} (copy)` here? for both of these setters